package.json

```diff
@@ -51,6 +52,7 @@
   },
   "scripts": {
     "docs": "./node_modules/apidoc/bin/apidoc -i src/ -o apidoc/",
+    "test:coverage": " NODE_ENV=test nyc --reporter=text mocha --recursive --exit || true",
```

Comment:
Do we want to enforce consistency in naming conventions here? For instance, any `lint` command is running `eslint.js` through `node`. Any `start` command is running `src/index.js` through `node`. Should any `test` command run `mocha`? The command below would become `test:base` or `test:run` instead of `start:test`.

Reply:
that could be a good change. I do admit I haven't been liking having to say start when running tests